<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWinClass</class>
 <widget class="QWidget" name="loginWinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>292</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>414</width>
    <height>292</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>414</width>
    <height>292</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TickTrader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tradewidget.qrc">
    <normaloff>:/image/images/title.png</normaloff>:/image/images/title.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="tradewidget.qrc">:/image/images/title.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>107</x>
        <y>175</y>
        <width>210</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>30</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogin">
      <property name="geometry">
       <rect>
        <x>107</x>
        <y>250</y>
        <width>210</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QWidget" name="statusBar" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>860</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
      <widget class="QLabel" name="loginStatus">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>验证中</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>221</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>55</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="servBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>107</x>
        <y>210</y>
        <width>210</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>30</height>
       </size>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>电信</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7*24</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 26pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>TickTrader</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUser">
      <property name="geometry">
       <rect>
        <x>107</x>
        <y>140</y>
        <width>210</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>210</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>账号</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tradewidget.qrc"/>
 </resources>
 <connections/>
</ui>
